controllers/administration.js

```diff
@@ -2173,7 +2173,9 @@ router.all(
 			$skip: itemsPerPage * (currentPage - 1),
 		};
 		query = Object.assign(query, filterQuery);
-
+		if (Object.values(query).length > 3) { // only reverse the order if the sorting key has been added to the object
+			query['$sort[displayName]'] == 1 ? query['$sort[displayName]'] = -1 : query['$sort[displayName]'] = 1
```

Comment:
This doesn't work if you sort by something different than displayName.
Maybe you can do something like:
```
const sort = Object.keys(query).find(a => a.startsWith("$sort"))
if (sort){
  query[sort] = query[sort] === 1 ? -1 : 1
}
```
Still hacky but would work for all sort attributes.

Reply:
This solution will only work in Administration, don't we want to sort classes in other places as well? Like in course edit -> classes dropdown. Wouldn't it be better to implement it server-side then?

Reply:
As far as I understood it, the problem only occurred at this one point. 🤷‍♂️